python/tk_unreal/unreal_sg_engine.py

```diff
@@ -64,14 +65,14 @@ def show_shotgunpanel_with_context():
                 app = engine.apps["tk-multi-shotgunpanel"]
                 entity_type, entity_id = self._get_context(engine)
                 if entity_type:
-                    return lambda : app.navigate(entity_type, entity_id, app.DIALOG)
+                    return lambda: app.navigate(entity_type, entity_id, app.DIALOG)
```

Comment:
It's ok to return lambdas as callbacks just consider that debugging issues when having lambdas is hard due that they don't give explicit info on the traceback.